native/c/zcli.hpp

```diff
@@ -526,10 +519,10 @@ int ZCLI::run(int argc, char *argv[])
   }
 
   // attempt to get a group
-  ZCLIGroup &group = get_group(argv[CLI_GROUP_ARG]);
+  ZCLIGroup *group = get_group(argv[CLI_GROUP_ARG]);
 
   // show main help if unknown group
-  if (0 == group.get_verbs().size())
+  if (0 == group->get_verbs().size())
```

Comment:
we should do a `group != nullptr` sanity check before trying `group->get_verbs()`

Reply:
fixt